minesweeper.py

```diff
@@ -340,7 +340,7 @@ def get_avg_time(highscore_dict):
     for highscore_dict in sorted(json_dict["high_scores"], key=get_highscore_data):
         seconds = round(highscore_dict["time"])
         if seconds >= 60:
-            format_time = f"{seconds / 60} min & {seconds % 60} sec"
+            format_time = f"{int(seconds / 60)} min & {seconds % 60} sec"
```

Comment:
So in this sense, you're just adding back "int" which was in my code previously.  87 seconds without int would become "1.45 min & 27 sec" whereas with "int" it becomes "1 min & 27 sec"

Reply:
Right. I'm just adding an `int()` that was accidentally deleted.